scripts/coreos-installer-service

```diff
@@ -97,7 +100,7 @@ for item in "${cmdline[@]}"; do
             firstboot_args+="${item} "
         fi
     done
-    for param in "${PERSIST_S390X_PARAMS[@]}"; do
+    for param in "${PERSIST_CONSOLE_PARAMS[@]}" "${PERSIST_S390X_PARAMS[@]}"; do
```

Comment:
@jlebon in https://github.com/coreos/coreos-installer/pull/605#discussion_r890516297:
> For `console` kargs specifically, we should probably also delete all previous platform-default `console` kargs before persisting those from the live environment. Although the last one wins, non-last console kargs still have an effect, and it wouldn't be correct to say we're exactly matching the console kargs of the live environment.
> 
> coreos-installer doesn't support deleting kargs by key only, but with a `--copy-console-kargs`, it could learn to delete all the metal kargs (and not append any platform kargs) if it picks up any `console` kargs from the live environment.